### DISCUSSION TITLE
Offline installation of spacy news package

### DISCUSSION THREAD

**Discussion by mahmoodn** [explosion/spaCy#9245] [Help: Installation] [answered]
Hi
I use the command `python3 -m spacy download de_core_news_sm` to fetch a package from internet. I want to know if I can download the package offline and manually install that. Any thought?

**Comment by adrianeboyd** [comment, marked as answer]
Yes, see: https://spacy.io/usage/models#download-pip

**Comment by mahmoodn** [comment]
OK Thanks.
